lib/cocoapods/installer.rb

```diff
@@ -1,4 +1,5 @@
 require 'active_support/core_ext/string/inflections'
+require 'concurrent/executor/fixed_thread_pool'
```

Comment:
we can move this `require` statement only if the installation option is enabled.

Reply:
Nice, didn't know you could do that. Changed